src/components/ArticleList.jsx

```diff
@@ -3,24 +3,37 @@ import ArticlePreview from "./ArticlePreview";
 import { fetchAllArticles } from "./utilities/requests";
 import "./style/articles.css";
 import { indicateLoading } from "./utilities/utilityFunctions";
+import SortBy from "./SortBy";
+import OrderBy from "./OrderBy";
 
 export default function ArticlesList({ topic }) {
   const [articles, setArticles] = useState([]);
   const [isLoading, setIsLoading] = useState(false);
+  const [sortBy, setSortBy] = useState("votes");
+  const [orderBy, setOrderBy] = useState("DESC");
 
   useEffect(() => {
     setIsLoading(true);
-    fetchAllArticles(topic).then((articles) => {
-      setIsLoading(false);
-      setArticles(articles);
-    });
-  }, [topic]);
+    fetchAllArticles(topic, `?sort_by=${sortBy}&order=${orderBy}`).then(
```

Comment:
You can pass your sort_by and order as params queries as well - into the params object in axios. Makes it a bit cleaner